src/peergos/server/storage/IpfsDHT.java

```diff
@@ -45,16 +44,40 @@ public IpfsDHT(MultiAddress ipfsAddress) {
     }
 
     @Override
-    public CompletableFuture<List<Multihash>> put(PublicKeyHash owner, PublicKeyHash writer, List<byte[]> signatures, List<byte[]> blocks) {
-        return put(writer, signatures, blocks, "cbor");
+    public CompletableFuture<TransactionId> startTransaction(PublicKeyHash owner) {
+        // TODO Implement once IPFS has
+        return CompletableFuture.completedFuture(new TransactionId(Long.toString(System.currentTimeMillis())));
```

Comment:
The resolution of timer in the VM can be very course. Recommend random number even for toy implementation

Reply:
This could be a constant and it wouldn't matter. It will be removed once IPFS implements it.